homeworks/test17/task4/task4.py

```diff
@@ -0,0 +1,24 @@
+def add_one(digits):
+    try:
+        i = len(digits) - 1
+        while i >= 0 and digits[i] == 9:
+            digits[i] = 0
+            i -= 1
+
+        if i >= 0:
+            digits[i] += 1
+            return digits
+        else:
+            return [1] + digits
+    except IndexError:
+        return None
```

Comment:
Not the best way to handle an exception, because we will never know about this type of issue
```suggestion
    except IndexError as e:
        logger.error(e)
```